File: test/fixtures/ocaml/multiline_post_with_data.ml
open Lwt
open Cohttp
open Cohttp_lwt_unix

let uri = Uri.of_string "http://localhost:28139/echo/html/" in
let headers = Header.init ()
  |> fun h -> Header.add h "Origin" "http://fiddle.jshell.net"
  |> fun h -> Header.add h "Content-Type" "application/x-www-form-urlencoded" in
let body = Cohttp_lwt.Body.of_string "msg1=value1&msg2=value2" in
Client.call ~headers ~body `GET uri
>>= fun (resp, body) ->
  (* Do stuff with the result *)

File: test/fixtures/julia/multiline_post_with_data.jl
using HTTP

headers = Dict(
    "Origin" => "http://fiddle.jshell.net",
    "Content-Type" => "application/x-www-form-urlencoded"
)

body = Dict(
    "msg1" => "value1",
    "msg2" => "value2"
)

resp = HTTP.get("http://localhost:28139/echo/html/", headers, body)
